.github/workflows/static-analysis.yaml

```diff
@@ -22,6 +22,13 @@ jobs:
         with:
           version: '3.20.3'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - run: |
+          set -euo pipefail
+
+          cargo install marker --version 0.9.0
```

Comment:
Is there a markdown linter written in golang and integrated with https://github.com/golangci/golangci-lint ?

Reply:
I didn't see golang version for markdown linter. And I don't think golangci-lint can cover this part because it's out of scope for golangci-lint.